### DISCUSSION TITLE
How to let window auto size with browser window size?

### DISCUSSION THREAD

**Discussion by kitepad** [emilk/egui#416] [Q&A]
I check out egui_template and compile to web. But ui canvas is fixed size in browser window, can not claim all width even browser window size is bigger.

**Comment by emilk** [comment]
Check out https://docs.rs/epi/0.12.0/epi/trait.App.html#method.max_size_points

**Reply by maxwase** [reply]
That is really counter-intuitive, isn't it? I can't understand why my app maximizes correctly on the native platform, but needs 
```rust
fn max_size_points(&self) -> Vec2 {
    vec2(f32::MAX, f32::MAX)
}
```
on web.
Is this an expected behavior? Maybe we should make default implementation? 

P.S.
Main egui demo works perfectly at `https://emilk.github.io/egui/index.html`, but when compiled locally, its size is incorrect.
![pages](https://user-images.githubusercontent.com/23321756/148468603-072e74a3-15ae-42ae-9347-c41e6fbb9ef9.png) 
![local](https://user-images.githubusercontent.com/23321756/148468700-a7fb0ecc-48c2-487f-a780-658ad4fc7426.png)

**Reply by emilk** [reply]
It's not ideal, but if we fill the full width WebGL gets _really_ slow in firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1010527#c0
